src/app/component/dashboard/customers/customers.component.html

```diff
@@ -144,15 +151,15 @@ <h2 class="font-semibold">
             <div class="col-sm-6" style="margin-bottom: 3vh">
               <div class="row mgbt-xs-0">
                 <kendo-textbox-container floatingLabel="{{ language.attention }}">
-                  <textarea kendoTextArea [(ngModel)]="data.attention" [autoSize]="true" name="attention"
+                  <textarea kendoTextArea [(ngModel)]="data.attention" (change)="inputChange()" [autoSize]="true" name="attention"
```

Comment:
This solution looks fine, but maybe it was easier and faster to do something like this:

- When opening modal for edit, for example, you may save the current state of the data object.
- If the user clicks on ESC or close, after that you may compare the old state of a data object and the current state and if you have some differences you can show an attention message